forge/housekeeper/tasks/expireTokens.js

```diff
@@ -1,9 +1,13 @@
 const { Op } = require('sequelize')
 
+const randomInt = (min, max) => { return min + Math.floor(Math.random() * (max - min)) }
```

Comment:
Slightly pedantic but my OCD is twitching here Nick.  In the original version, `min` is inclusive but `max` is exclusive.  i.e. in its current form, `randomInt(/*min*/0, /*max*/1)` will ALWAYS return `0`. 

Can I propose both min and max are inclusive?

Suggestion: 

```suggestion
const randomInt = (min, max) => Math.floor(Math.random() * (max - min + 1) + min)
```

Reply:
We use this pattern in a couple of the timertasks with this function copied between them. If we change it in one, we'll need to change them all so as not to have inconsistent implementations.

Reply:
https://github.com/FlowFuse/flowfuse/pull/3327/commits/9bb33fa445c23daa24226ecce5df43bb1054d309